### ISSUE TITLE
`JSXIdentifier` => `JSXOpeningElement ` causes regression.

### ISSUE THREAD

**Issue by AlexKVal** [yannickcr/eslint-plugin-react#73]
With this https://github.com/yannickcr/eslint-plugin-react/commit/3fc121bcf29 
I've got 
```
docs/src/Root.js
  2:7  warning  Router is defined but never used  no-unused-vars
```
with the code: [react-bootstrap/docs/src/Root.js#L2](https://github.com/AlexKVal/react-bootstrap/blob/master/docs/src/Root.js#L2)
```js
import Router from 'react-router';
...
const Root = React.createClass({
...
render() {
  return (
     <html>
...
       <Router.RouteHandler />
...
```

**Comment by AlexKVal** [comment]
A quick look exposed that the failure is caused by:

with this code I've got warning
```js
<Router.RouteHandler />
```
and with this code the warning disappears
```js
<Router />
```

**Comment by yannickcr** [comment]
Thanks. I found the problem and I will release a quick fix.
